src/main/java/edu/ksu/canvas/attendance/services/CanvasApiWrapperService.java

```diff
@@ -163,6 +171,41 @@ public void deleteAssignment(String courseId, String canvasAssignmentId, OauthTo
         assignmentWriter.deleteAssignment(courseId, canvasAssignmentId);
     }
 
+    public AssignmentOverride createAssignmentOverride(OauthToken oauthToken, Integer assignmentId,
```

Comment:
In the good spirit of attendance being the 'Golden child' application (I think that was the term we coined when @zoglmannk first wrote it), there should probably be a test added for this functionality.